composer/loggers/mlflow_logger.py

```diff
@@ -47,9 +47,13 @@ class MLFlowLogger(LoggerDestination):
             (default: ``10``).
         model_registry_prefix (str, optional): The prefix to use when registering models.
             If registering to Unity Catalog, must be in the format ``{catalog_name}.{schema_name}``.
-            (default: empty string)
+            (default: `''`)
         model_registry_uri (str, optional): The URI of the model registry to use. To register models
             to Unity Catalog, set to ``databricks-uc``. (default: None)
+        synchronous (bool, optional): Whether to log synchronously. If ``True``, Mlflow will log
```

Comment:
when would anyone ever want synchronous? I think we should remove this in a future PR

Reply:
yea it's more like a safety flag, I will make a PR to remove it once we are confident with the functionality.